api/v1alpha1/condition_consts.go

```diff
@@ -9,15 +9,33 @@ const (
 	ReadyCondition ConditionType = "Ready"
 )
 
+const (
+	// VirtualMachinePreReqReadyCondition is set on VirtualMachines when all of a VirtualMachine's prerequisites are satisfied.
+	// For now, this check only validates that the VirtualMachineClass specified in the VirtualMachineSpec exists on the API
```

Comment:
I don't think we need to call out "on the API server"; just exists is sufficient.

I'd reword this so we don't have to constantly update it, and for it to become out of date when we do. Say something along the lines of the reasons below are satisfied.